src/common/redis_module/ray_redis_module.cc

```diff
@@ -908,6 +910,74 @@ int TableTestAndUpdate_RedisCommand(RedisModuleCtx *ctx,
   return result;
 }
 
+/* send notification to the subscribers */
+int sendObjectNotification(RedisModuleCtx *ctx,
+                           RedisModuleString *object_id,
+                           RedisModuleKey *table_key, 
+                           bool deleted) {
```

Comment:
I don't see why we need the `deleted` argument.

In both cases, we should send the list of managers, and in the "deleted" case, that list is empty, right?

Reply:
During deploying this feature, I had encountered some problems in ObjectTableRemove_RedisCommand, E.g protocol checking failed. So I add this "deleted" param. The document about this flag:
 * @param deleted The flag which indicates that the object has been removed. Maybe
 *        some entries about this object have been deleted. In this case, some fields
 *        of the notification protocol message will be filled in NULL or zero, instead of
 *        getting some meta in db (E.g data size or object manager list).